starwars/src/App.js

```diff
@@ -1,19 +1,32 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+import CharacterList from "./components/CharacterList";
+import CharacterCard from "./components/CharacterCard";
+import styled from "styled-components";
 
 const App = () => {
   // Try to think through what state you'll need for this app before starting. Then build out
   // the state properties here.
 
-  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a 
+  // Fetch characters from the star wars api in an effect hook. Remember, anytime you have a
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
 
   return (
     <div className="App">
-      <h1 className="Header">React Wars</h1>
+      <StyledHeader>
+        {" "}
+        <h1 className="Header">React Wars</h1>
+        <CharacterList />
+        <CharacterCard />
+      </StyledHeader>
     </div>
   );
-}
+};
 
+const StyledHeader = styled.h1`
+  text-align: center;
+  margin: 0 auto;
+  font-size: 2rem;
+`;
```

Comment:
We went over this during our review, but since we're using `StyledHeader` as a container component, we can use a different element like `styled.div`